lib/new_relic/agent/configuration/default_source.rb

```diff
@@ -2188,6 +2188,30 @@ def self.notify
           :transform => DefaultSource.method(:convert_to_constant_list),
           :description => 'Specify a list of exceptions you do not want the agent to strip when [strip_exception_messages](#strip_exception_messages-enabled) is `true`. Separate exceptions with a comma. For example, `"ImportantException,PreserveMessageException"`.'
         },
+        # Agent Control
+        :'agent_control.fleet_id' => {
+          :default => nil,
+          :allow_nil => true,
+          :public => true,
```

Comment:
if these new configs are something that will be set by fleet control, we don't want customers setting it themselves right? Should they all be set to not public then?

Reply:
Also it looks like these are all env variable only, so maybe we should add that to the description?

Reply:
@tannalynn - How does this rewording sound for the last sentence of each of these configs? 

`This configuration is set by agent control, or one of its components, as an environment variable prior to agent startup.`

Reply:
I've updated the configs to reference the environment variable. I'm going to resolve this, but feel free to reopen if you'd like them worded in a different way.